cl/beacon/handler/handler.go

```diff
@@ -168,6 +174,7 @@ func (a *ApiHandler) init() {
 					r.Get("/sync_committee_contribution", http.NotFound)
 					r.Post("/contribution_and_proofs", http.NotFound)
 					r.Post("/prepare_beacon_proposer", http.NotFound)
```

Comment:
Let's remove this endpoint definition?

Reply:
this is just a placeholder

Reply:
But you already implemented it at line 177?!

Reply:
good catch